packages/guui/components/Caption/Caption.tsx

```diff
@@ -79,7 +83,7 @@ export const Caption: React.FC<{
                         <span className={iconStyle}>
                             <TriangleIcon />
                         </span>
-                        {getCaptionHtml()}
+                        {getCaptionHtml()} {displayCredit && credit}
```

Comment:
Only question is can the credit contain HTML?

Reply:
There was [this](https://github.com/guardian/flexible-content/pull/1808) (never went in), but not inconceivable in the future and being asked for from time to time. 
Practically, not possible in the current state of things, though.

Reply:
> Only question is can the credit contain HTML? If so, we may need to enable it/prevent escaping.

Yep, it still works:
<img width="788" alt="Screen Shot 2019-07-17 at 09 33 59" src="https://user-images.githubusercontent.com/2051501/61360325-21e87300-a876-11e9-8395-af54f16a84c8.png">

Reply:
> There was [this](https://github.com/guardian/flexible-content/pull/1808) (never went in), but not inconceivable in the future and being asked for from time to time.
> Practically, not possible in the current state of things, though.

Thanks Mat.

Reply:
Cool thanks both!